src/browser.js

```diff
@@ -148,35 +204,20 @@ function save(namespaces) {
  */
 
 function load() {
-  try {
-    return exports.storage.debug;
-  } catch(e) {}
-
-  // If debug isn't set in LS, and we're in Electron, try to load $DEBUG
-  if (typeof process !== 'undefined' && 'env' in process) {
-    return process.env.DEBUG;
+  if (exports.storage) {
+    exports.storage.get('debug', exports.enable);
```

Comment:
I don't follow you here at all. There's no `enabled` and there's no `exports.names`. Care to rephrase?